### ISSUE TITLE
[Question] Your Working Config

### ISSUE THREAD

**Issue by barrett-ruth** [lewis6991/spellsitter.nvim#74]
Hello, I really want to use spellsitter. However, the `[s` and `]s` navigation simply does not work for me. I was on nightly, so I switched to 0.7. No dice. I see lots of issues in here in which people share their config - can you share yours? Your `spellfile`, setup config, everything. The highlighting correctly works for me but the nav function seems to just not work even with an empty spellfile (the cursor stays still).

**Comment by barrett-ruth** [comment]
It seems that `additional_vim_regex_highlighting = true` was messing up my config. Could you explain why this is so (and then feel free to close)?

**Comment by lewis6991** [comment]
Updated README in d8bdd4923c1bc8fc9e66e94ca65544a16ffb96b2

**Comment by barrett-ruth** [comment]
Yes - of course I meant to set that value to false not true. My mistake. Thanks for the update.
